troposphere/static/js/components/projects/detail/resources/InstanceBatchActionButtons.jsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import Backbone from "backbone";
+// NOTE raven-js included during evaluation
+// - may be removed after ATMO-981 resolved
```

Comment:
I would prefer the raven stuff to be removed (as was probably the intention). Including it like below makes it more coupled to our codebase and less of an optional thing.